var/spack/repos/builtin/packages/verrou/package.py

```diff
@@ -0,0 +1,115 @@
+    a subset of Monte Carlo Arithmetic, performing only output randomization
+    through random rounding.
+    """
+
+    homepage = "https://github.com/edf-hpc/verrou"
+    url      = "https://github.com/edf-hpc/verrou.git"
+
+    version('master',
+            git='https://github.com/edf-hpc/verrou.git',
+            branch='master')
+
+    version('2.0.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='a614ad695d060abdfcd3cec11022d6f044a0980d')
+    version('1.1.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='d69787e89507519e14f5ec44150c1882930f692e')
+    version('1.0.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='519dd9a83738f72da8446322e9c403e608940b7a')
+    version('0.9.0',
+            git='https://github.com/edf-hpc/verrou.git',
+            commit='ebabfcf7f4e9a02b130e9819ba6c19e0450eb73d')
+
+    resource(name='valgrind-3.13.0',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='1378ec95f22235e0a8c972cf1cd0abef0b9610d4',
+             when='@1.1.0:,master')
+    resource(name='valgrind-3.12.0',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='36923ab298c8d2791d6c86b6cd5c0667c07449f6',
+             when='@1.0.0:1.0.999')
+    resource(name='valgrind-3.10.1',
+             git='git://sourceware.org/git/valgrind.git',
+             commit='cfc3175a6eb132a5600fcf705f3fa326d71483bf',
+             when='@0.9.0:0.9.999')
+
+    variant('fma', default=True,
+            description='Activates fused multiply-add support for Verrou')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('m4', type='build')
+
+    def patch(self):
+        # The current setup gives us the verrou source tree, with a "valgrind"
+        # subdirectory. But we want the reverse layout. Let's fix this.
+        verrou_files = os.listdir('.')
+        verrou_files.remove('valgrind')
+        os.mkdir('verrou')
+        for name in verrou_files:
+            os.rename(name, os.path.join('verrou', name))
+        for name in os.listdir('valgrind'):
+            os.rename(os.path.join('valgrind', name), name)
+        os.rmdir('valgrind')
+
+        # Once this is done, we can patch valgrind
+        which('patch')('-p0', '--input=verrou/valgrind.diff')
+
+        # Autogenerated perl path may be too long, need to fix this here
+        # because these files are used during the build.
+        for link_tool_in in glob.glob('coregrind/link_tool_exe_*.in'):
+            filter_file('^#! @PERL@',
+                        '#! /usr/bin/env perl',
+                        link_tool_in)
+
+    def configure_args(self):
+        spec = self.spec
+        options = ['--enable-only64bit']
+
+        if spec.satisfies('+fma'):
+            options.append('--enable-verrou-fma')
```

Comment:
More robust would be:
```
options.append('--{}-verrou-fma'.format('enable' if 'fma' in spec else 'disable'))
```
or if you need to save characters:
```
options.append('--%s-verrou-fma' % ('enable' if 'fma' in spec else 'disable'))
```
This way you do not rely on defaults in the autotools *configure* script, which might change over time.

Reply:
Yay, we actually have docs for this now!
http://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#addings-flags-to-configure

Reply:
Done. Nice catch!